BSS_Villager_Name-Title-Comment-ChatText1-ChatText2-ChatText3.csv

```diff
@@ -1299,30 +1298,30 @@ Old Witch
+Cupid	Lost Eros	San Joel 2019 Pink Valentine's Day	Where does Cupid's arrow go?	The arrow goes to the boy's heart!	The boy's heart...pumping...lover's whisper...
+Cupid	Lost Eros	Oasis 2019 Pink Valentine's Day	Where does Cupid's arrow go?	The arrow goes to the girl's heart!	the girl's heart...pumping...lover's whisper...
 Cupid	Lost Eros	Beginner's Village 2010 Pink Valentine's Day	I...I want...	I want you...I want your...I want your love...	Why don't you come over...
-Lynas	Priestess	Holy Crater 2010 Pink Valentine's Day	Why do you stare me like that?	Even though I am born beautiful, I'm still easy going.	Stop staring...I know its hard.
-Lynas	Priestess	Thunder Hill 2010 Pink Valentine's Day	Why don't you look at me?	Aren't you afraid of being yelled at? Think about why!	Stop staring at me like a piece of meat!
-Lynas	Priestess	San Joel 2010 Pink Valentine's Day	Why do you stare me like that?	Even though I am born beautiful, I'm still easy going.	Stop staring...I know its hard.
-Lynas	Priestess	oasis 2010 Pink Valentine's Day	Why don't you look at me?	Aren't you afraid of being yelled at? Think about why!	Stop staring at me like a piece of meat!
-Lynas	Priestess	Beginner's Village 2010 Pink Valentine's Day	I used to sneeze for no reason every Valentine's day.	Maybe it's another pursuer's thinking of me.	Or is it another girl's jealous of me?
-Cupid	Lost Eros	Holy Crater 2010 white Valentine's day	(scream) ah ah!	(scream) what happened to you? ah ah!	
-Cupid	Lost Eros	Thunder Hill 2010 white Valentine's day	(scream) Ahh!	(scream) what happened to you? ah ah!	
-Cupid	Lost Eros	San Joel 2010 white Valentine's day	(scream) Ahh!	(scream) what happened to you? ah ah!	
-Cupid	Lost Eros	oasis 2010 white Valentine's day	(scream) Ahh!	(scream) what happened to you? ah ah!	
-Cupid	Lost Eros	Beginner's Village 2010 white Valentine's day	(scream) Ahh!	(scream) what happened to you? ah ah!	
-Cupid	Lost Eros	trail field 2010 lover's ending song	If this is what love is about...	Should I leave or stay?	
-Cupid	Lost Eros	Holy Meteor Plateau 2010 lover's ending song	If this is what love is about...	Should I leave or stay?	
-Jack Foberts	Fugitive	Meteor Pit 2009 Halloween npcp0233	Ghost Castle�K	Hero!  I have some information I need to tell you!  Come over here quickly!	
-Jack Foberts	Fugitive	Thunder Hill 2009 Halloween npcp0233	Ghost Castle�K	Hero!  I have something I need to tell you!  Come over here quickly!	
-Abnormal World Stele	Abnormal World Passage	Meteor Pit 2009 Halloween npcp0233			
-Abnormal World Stele	Abnormal World Passage	Thunder Hill 2009 Halloween npcp0233			
-Shadow of Jack	Projection from the Abnormal World	Ghost Castle 2009 Halloween npc0233	Ghost Castle�K	Hero!  I have some information I need to tell you!  Come over here quickly!	
+Lynas	Priestess	Holy Crater 2019 Pink Valentine's Day	Why do you stare me like that?	Even though I am born beautiful, I'm still easy going.	Stop staring...I know its hard.

```

Comment:
the year should stay 2010 and 2009 since its part of the quest name and would distinguish between the 2009 quest and a quest that was created this year or one in the future